sdcm/logcollector.py

```diff
@@ -1254,12 +1254,12 @@ def __init__(self, test_id=None, test_dir=None, params=None):
         self.sct_set = []
         self.pt_report_set = []
         self.cluster_log_collectors = {
-            SSTablesCollector: self.db_cluster,
             ScyllaLogCollector: self.db_cluster,
+            SCTLogCollector: self.sct_set,
```

Comment:
Please, add also "k8s log collector"  to be after the SCT and before the monitoring. It is also fast and valueable to have earlier.

Reply:
Actually, the monitor log collection stayed before as it was. Ok, merged anyway...